common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -485,6 +481,16 @@
   "feature.wynntils.emeraldPouchHotkey.multipleFilled": "You have multiple filled emerald pouches in your inventory.",
   "feature.wynntils.emeraldPouchHotkey.name": "Emerald Pouch Hotkey",
   "feature.wynntils.emeraldPouchHotkey.noPouch": "You do not have an emerald pouch in your inventory.",
+  "feature.wynntils.experienceGainMessage.combat.description": "Shows combat Xp gain messages as a game update notification.",
+  "feature.wynntils.experienceGainMessage.combat.name": "Show Combat Xp Messages",
+  "feature.wynntils.experienceGainMessage.description": "Shows experience gain messages as a game update notification.",
+  "feature.wynntils.experienceGainMessage.filterChat.description": "Should xp gain messages be hidden from chat?",
+  "feature.wynntils.experienceGainMessage.filterChat.name": "Filter Chat Messages",
+  "feature.wynntils.experienceGainMessage.name": "Experience Gain Messages",
+  "feature.wynntils.experienceGainMessage.professions.description": "Shows profession Xp gain messages as a game update notification.",
+  "feature.wynntils.experienceGainMessage.professions.name": "Show Profession Xp Messages",
+  "feature.wynntils.experienceGainMessage.secondDelay.description": "How long (in seconds) should a XP gain message collect XP before it is a separate message?",
+  "feature.wynntils.experienceGainMessage.secondDelay.name": "XP Gain Message Consolation Time",
```

Comment:
Can you make these all use "XP" for consistency, after that everything should be good to go